.github/workflows/publish.yml

```diff
@@ -1,8 +1,10 @@
 name: Publish Bottles
 on:
   pull_request:
-    paths:
+    paths-ignore:
       - Formula/phylum.rb
+      - README.md
+      - .github/workflows/publish.yml
```

Comment:
Seems much simpler to delete the trigger or the entire workflow....

Reply:
The workflow has been deleted.